tests/unit/addon-tree-test-node.js

```diff
@@ -29,7 +35,11 @@ describe('Addon', function() {
       process.env.EMBER_ENV = 'production';
       var addonTree = getMirageAddon().treeFor('addon');
 
-      expect(addonTree.inputTrees.length).to.be.equal(0);
+      if (addonTree._inputNodes) {
+        expect(addonTree._inputNodes.length).to.equal(0);
+      } else {
+        expect(addonTree._inputNodes).to.be.undefined;
+      }
```

Comment:
This is very strange, but goes along with the behavior you and I were seeing early on master (where `length` was undefined on Mac, but not on Travis' Ubuntu version).